### ISSUE TITLE
Fedora 37: SIGSEGV when loading track

### ISSUE THREAD

**Issue by uklotzde** [mixxxdj/mixxx#11076]
### Bug Description

Still on Qt 5.15.6, updating to Qt 5.15.7 blocked by some RPM Fusion packages.

```
#0  0x00007ffff34ab53b in QCoreApplicationPrivate::sendPostedEvents(QObject*, int, QThreadData*) () at /lib64/libQt5Core.so.5
#1  0x00007ffff34f9537 in postEventSourceDispatch(_GSource*, int (*)(void*), void*) () at /lib64/libQt5Core.so.5
#2  0x00007ffff5818cbf in g_main_context_dispatch () at /lib64/libglib-2.0.so.0
#3  0x00007ffff586e598 in g_main_context_iterate.constprop () at /lib64/libglib-2.0.so.0
#4  0x00007ffff5815f40 in g_main_context_iteration () at /lib64/libglib-2.0.so.0
#5  0x00007ffff34f902a in QEventDispatcherGlib::processEvents(QFlags<QEventLoop::ProcessEventsFlag>) () at /lib64/libQt5Core.so.5
#6  0x00007ffff34a6c1a in QEventLoop::exec(QFlags<QEventLoop::ProcessEventsFlag>) () at /lib64/libQt5Core.so.5
#7  0x00007ffff34aece2 in QCoreApplication::exec() () at /lib64/libQt5Core.so.5
#8  0x00000000004c4c34 in (anonymous namespace)::runMixxx(MixxxApplication*, CmdlineArgs const&) (pApp=0x7fffffffd5d8, args=<optimized out>)
    at /home/uk/volumes/Build/cpp/mixxx/src/main.cpp:72
#9  main(int, char**) (argc=<optimized out>, argv=<optimized out>) at /home/uk/volumes/Build/cpp/mixxx/src/main.cpp:200
```

### Version

main

### OS

Fedora 37

**Comment by uklotzde** [comment]
Here we go again, this time with a Clang 15 build.

```
Process 76745 stopped
* thread #1, name = 'mixxx', stop reason = signal SIGSEGV: invalid address (fault address: 0x8)
    frame #0: 0x00007ffff6454c67 libQt5Widgets.so.5`QStyleSheetStyle::renderRule(QObject const*, int, unsigned long long) const + 359
libQt5Widgets.so.5`QStyleSheetStyle::renderRule:
->  0x7ffff6454c67 <+359>: cmpl   0x8(%rdx), %r13d
    0x7ffff6454c6b <+363>: jne    0x7ffff6454c58            ; <+344>
    0x7ffff6454c6d <+365>: cmpl   %r12d, 0xc(%rdx)
    0x7ffff6454c71 <+369>: jne    0x7ffff6454c58            ; <+344>
  thread #14, name = 'mixxx', stop reason = signal SIGSEGV: address access protected (fault address: 0x7fff4c597000)
    frame #0: 0x00007ffff69b985d libSoundTouch.so.2`soundtouch::InterpolateCubic::transposeStereo(float*, float const*, int&) + 301
libSoundTouch.so.2`soundtouch::InterpolateCubic::transposeStereo:
->  0x7ffff69b985d <+301>: movlps %xmm2, -0x8(%rsi,%rax,8)
    0x7ffff69b9862 <+306>: movsd  %xmm1, 0x18(%rdi)
    0x7ffff69b9867 <+311>: cmpl   %ecx, %r8d
    0x7ffff69b986a <+314>: jg     0x7ffff69b9780            ; <+80>
```

**Comment by uklotzde** [comment]
Duplicate of #11049.
